uptane-standard.md

```diff
@@ -240,28 +277,222 @@ The Time Server SHALL expose a public interface allowing primaries to communicat
 
 ## In-vehicle implementation requirements
 
+An Uptane-compliant ECU SHALL be able to download and verify the time, metadata, and image binaries before installing a new image.
+
+Each ECU in a vehicle receiving over-the-air updates is either a primary or a secondary ECU. A primary ECU collects and delivers to the Director vehicle manifests ({{vehicle_version_manifest}}) containing information about which images have been installed on ECUs in the vehicle. It also downloads and verifies the latest time, metadata, and images for itself and for its secondaries. A secondary ECU downloads and verifies the latest time, metadata, and images for itself from its associated primary ECU. It also sends signed information about its installed images to its associated primary.
+
+All ECUs MUST verify image metadata as specified in {{metadata_verification}} before installing an image or making it available to other ECUs. A primary ECU MUST perform full verification ({{full_verification}}). A secondary ECU SHOULD perform full verification if possible, and MUST perform full verification if it is safety-critical. If it is not safety-critical, it MAY perform partial verification ({{partial_verification}}) instead.
+
+### Build-time prerequisite requirements for ECUs
+
+For an ECU to be capable of receiving Uptane-secured updates, it MUST have the following data provisioned at the time it is manufactured or installed in the vehicle:
+
+1. The latest copy of required Uptane metadata at the time of manufacture or install.
+    * Partial verification ECUs MUST have the root and targets metadata from the director repository.
+    * Full verification ECUs MUST have a complete set of metadata from both repositories (root, targets, snapshot, and timestamp), as well as the repository map file.
```

Comment:
Perhaps this is a dumb question, but wouldn't just the root metadata be enough? I can't see how it would affect the security properties of the protocol. Advantage is that it would simplify the provisioning process at the expense of a little extra complexity to the client.

Reply:
My interpretation is that Uptane would consider initial provisioning as part of the "install" process, and so it would be out of scope to specify exactly how it's done. That is, provisioning the rest of the metadata using just the root metadata would be fine; it's just another way to ensure that the required metadata is there when the car rolls off the line.

Maybe we could add some text clarifying that?

Reply:
I was thinking that the process of provisioning on the production line and the process of sending a update will be handled by separate parts of the organization, so it might be easier if the provisioning part has the minimal amount stuff in it.

It might be possible to construct a '⊥' set of metadata that doesn't contain any images at all, and could be loaded as part of the provisioning process, however the client still has to be able to work without the other metadata, because when the root key is rotated the targets etc get deleted.

What is the advantage of including targets etc during provisioning?

Reply:
Sorry, what is the "repository map file"?

Reply:
@cajun-rat As with the other comment, can you raise this as an issue after merge?